vm-deploy.yaml

```diff
@@ -14,9 +14,6 @@ spec:
   schedulerName: autoscale-scheduler
   enableNetworkMonitoring: true
   enableSSH: true
-  tlsCertificateIssuer: "neon-ca-issuer"
-  tlsExpiration: 5m
-  tlsRenewal: 3m
```

Comment:
Ah, I see. we later changed the format, I guess I missed the example.

We could fix it to
```yaml
tls:
  certificateIssuer: "neon-ca-issuer"
  expireAfter: 5m
  renewBefore: 3m
  serverName: "example.local"
```

but I don't care much whether we just get rid of it either

Reply:
Thanks, I fixed the example to reflect the correct format